vpype/io.py

```diff
@@ -364,6 +365,8 @@ def write_svg(
         show_pen_up: add paths for the pen-up trajectories
         color_mode: "none" (no formatting), "layer" (one color per layer), "path" (one color
             per path)
+        no_basic_shapes: if false we use svg:path elements to write lines. Else we use 
```

Comment:
In addition to the param name change (and possibly language improvement), the doc string should add a note on the purpose of this feature (e.g faster loading times in inkscape).

Reply:
i added some comment about that